doc/TRIAGING.md

```diff
@@ -0,0 +1,146 @@
+
+If you read the description of an issue and don't even understand the terminology used, but know who on the team understands it, it is fine to ping that person right away. It is also fine to take time and get familiar with the terminology used, so use your judgement. If you have no idea whom to ping, ask the team on Slack.
+
+It is fine to immediately ping somebody if you suspect they might know the answer to the question asked right away.
+
+To be very clear, you're not expected to resolve any issue yourself by implementing a fix for example. However if you feel like there's a chance that it won't take long to resolve an issue, do try to do it yourself right away. But if you're not making progress within 30 minutes to an hour, ask somebody.
+
+Triaging isn't meant to compete with the regular planning process, so don't do any sizeable amount of work without this work going through proper prioritization as per our development routine.
+
+### Ownership
+
+Just pinging somebody isn't making progress. Make sure the person you've pinged responds and do feel free to bug them until they do. Once a week or two weeks should be reasonable for issues that are not high-priority. For high-priority ones raise the issue as soon as possible on Slack to the whole team and do it vocally.
+
+In general, once you've triaged an issue it becomes your responsibility to follow up on it, which includes not only pinging people, but also making sure that the work on the issue is being planned, put into a sprint and eventually actually performed. It is exclusively your responsibility to make sure that an issue triaged by you is tracked and that progress is more or less regularly made. This is irrespective of whether another person has taken on the triage role. Once you triage an issue, it's yours until some other person takes over it, which needs to be explicitly reflected in the issue thread.
+
+Every issue labeled as "status: needs action from the team" (more on that later) needs to have the "assignee" field filled with the name of the person who triaged the issue, so once you've triaged an issue, please fill the field with your name. You cannot fill the field with other's person name. This is because we want to ensure that people are fully aware of their responsibilities, as otherwise it's trivial to forget about an issue: one person triages it, assigns it to another person and feels like their job is done, while the other person isn't even aware that there's an issue assigned to them. People are also more keen to serve a duty that they've taken on themselves as opposed to being randomly assigned by somebody else.
+
+Note how issues are not assigned to those who are supposed to resolve them by submitting PRs for example. This is because for that we have our regular development process with sprints and tickets and whatnot, so there wouldn't be much sense in duplicating this information on GitHub.
+
```

Comment:
Not sure what you mean. Jira stuff is private and we're already supposed to reflect PRs in Jira (which I haven't been very good at...), so I'm not sure what else we can do here.

Reply:
I just meant to mention in the PR the GH issue it's trying to resolve, so that it will appear in the GH issue itself. I think we do already do it but it's maybe a good idea to explicitly encourage it in this document. I also think that even though Jira is private, it's helpful for our team to keep track if the Jira ticket is mentioned.

Reply:
Right, thanks, will do both.